app/assets/javascripts/application.js

```diff
@@ -30,6 +30,7 @@
 //= require templates
 //= require indicators
 //= require scorecards/datepicker_popup
```

Comment:
@hengsokly please consider remove datepicker_popup including it library if there is no any places keep using